### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by samlorlah**
# In this Project

- Linters was added
- Html file was created with an header
- Stylesheet was created to style the HTML header
- A Test.md file was created and gitignore was used to ignore it from being pushed to github

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Ismailco) [inline .gitignore]: - [x] `node_modules` folder should be in the `.gitignore` file, so git ignore it and wouldn't be pushed to your Github repo, kinky consider adding node_modules to this file

Comment (Ismailco) [inline .gitignore]: - [x] The Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly in the future project keep this in mind

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+
+# Project Name
```
Comment (Ismailco) [inline README.md:3]: - [x] [OPTIONAL] You can remove the project Name as it's just a placeholder

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+Built With
+Major languages - N/A
+Frameworks - N/A
+Technologies used
+Live Demo (if available)
+Live Demo Link
+
+# Project Name
+
+> Description the project.
```
Comment (Ismailco) [inline README.md:16]: - [x] [OPTIONAL] Kindly try to make your README more professional by adding every input in the right section

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @samlorlah,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (samlorlah) [reply .gitignore]: @Ismailco do you mean I should create another branch?

Comment (Ismailco) [reply .gitignore]: @samlorlah, No, you can ignore that for this PR but in the future project pay more attention to the naming of the branch

Comment (tufoinnkuo10) [inline .gitignore]: - [x] like the previous reviewer noted, your `node_modules` folder should be in the `.gitignore file`, so git ignore it and wouldn't be pushed to your Github repo, kinky consider adding node_modules to this file

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @samlorlah 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated


Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: .gitignore (hunk 1 above)
Comment (samlorlah) [reply .gitignore]: This was added in the last commit, just that I used a new branch which I quoted.

I have pushed it to this branch with node_modules added

**Review by zhadier — APPROVED** [review]
## Hi @samlorlah  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**